README.md

```diff
@@ -16,52 +16,53 @@ To add the driver to your project, add the following Maven dependency:
 </dependency>
 ```
 
-The class name for this driver is:
+Replace `${jdbc.version}` with the actual version of the driver. It's good practice to manage this version in a properties section of your pom.xml.
 
-```
-com.salesforce.datacloud.jdbc.DataCloudJDBCDriver
-```
+### Driver Class
 
-## Building the driver:
+The driver class name is: `com.salesforce.datacloud.jdbc.DataCloudJDBCDriver`
 
-Use the following command to build and test the driver:
+## Building the Driver
 
-```shell
+To build and test the driver, run:
+
+```bash
 mvn clean install
 ```
 
```

Comment:
I think we should move this down all the way to the end of this document? Our customers should never need to build the driver from source. Building the driver is only a task which our own developers (and other open-source contributors) will have to do